src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -263,14 +333,14 @@ export function secureMessageDecrypt64(encrypted64, privateKey64, publicKey64) {
         });
     });
 }
-export function string64(input) {
-    return Buffer.from(input).toString('base64');
-}
 /* Returns UUID in string value that corresponds to new comparator */
 export function comparatorInit64(data64) {
     if (data64 === "" || data64 === undefined || data64 === null) {
         throw new Error("Parameter data64 can not be empty");
     }
+    if (!isBase64(data64)) {
+        throw new Error("Parameter data64 is not base64 encoded");
+    }
```

Comment:
Also, why not put these checks into a single `convertInputBuffer()` function or something?

Reply:
Done.